### ISSUE TITLE
List packages in a table, not just a single row going down

### ISSUE THREAD

**Issue by vadi2** [Mudlet/mudlet-package-repo#82]
The packages are all listed in one row going down, instead of going left to right, next row, left to right.

I had a small scrap with bootstrap about this but couldn't make it flow just yet.

**Comment by Kebap** [comment]
Flowing items from one row to the next depending on viewport size and space between shared evenly around all items seems easily doable with css flexboxes here: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox

**Comment by vadi2** [comment]
Yep it's already a flexbox, see the Bootstrap grid that is used for this: https://getbootstrap.com/docs/4.0/layout/grid/
